### ISSUE TITLE
UserWarning: Implicit dimension choice for softmax has been deprecated

### ISSUE THREAD

**Issue by watson21** [ZhaoJ9014/face.evoLVe#178]
face.evoLVe/applications/align/get_nets.py:70: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  a = F.softmax(a)

specifically in this case, how many output neuron number of this model.
I using Onet, Pnet and Rnet
Btw, i don't see any flatten layer or linear layer in Onet, Pnet or Rnet
Thanks you
